index.html

```diff
@@ -0,0 +1,96 @@
+        </nav>
+        </header>
+    <section class="hero">
+                <div class="hero-cont">
+                    <h1 class="hero-text">Hello sharing world <br> 11th global<br> conference on global <br> <span id="span-head"> warming </span></h1>
+                    <p id="hero-p">The 11th Global Conference on Global Warming (GCGW-2023) is a multi–disciplinary international conference on all aspects of global warming including its causes, impacts and potential solutions. The conference aims to provide a forum for the exchange of technical information.</p>
+                    <h2>2015.10.15(THU) ~ 16(FRI)</h2>
+                    </div>
+    </section>
+     <!--main program-->
+                    <section class="main-program-background">
+                        <h2 class="main-program-h2">Main program</h2>
+                    <div class="main-program">
+                        <div class="cards">
+                            <img src="images/base/program_icon_01.png" alt="">
+                            <h2>Lecture</h2>
+                            <p>Listen to the speakers from various countries .</p>
+                        </div>
+                        <div class="cards">
+                            <img class="card-icon" src="images/base/program_icon_02.png" alt="">
+                            <h2>CC Exhibition</h2>
+                            <p>Appreciate various creations applying CC license of artists. </p>
+                        </div>
+                        <div class="cards">
+                            <img src="images/base/program_icon_03.png" alt="">
+                            <h2>Forum</h2>
+                            <p>Have the time to share your thoughts and opinions .</p>
+                        </div>
+                        <div class="cards">
+                            <img src="images/base/program_icon_04.png" alt="">
+                            <h2>Workshop</h2>
+                            <p>Try creating your own work using open source license .</p>
+                        </div>
+                        <div class="cards">
+                            <img src="images/base/program_icon_05.png" alt="">
+                            <h2>CC ignite</h2>
+                            <p>get opportunities to network with CC affiliates around the world.</p>
+                        </div>
+                    </div>
+                    <a class="main-program-btn" href="#">Join CC Global Summit 2023</a>
+                </section>    
+                    <!--featured speakers-->
+                    <h2 class="Feature-h2">Featured Speakers</h2>
+                    <div class="featuerd-speakers">
+                    </div>
+                    <a id="more-speaker" href="#">more <i class="fa-solid fa-chevron-down"></i></a>
+                    <!--the partners-->
+                    <div class="Partners">
+                        <p>Partners</p>
```

Comment:
- [ ] Kindly make sure to add a separator to this section header and style the header properly (font, color) so your design looks identical to the provided design

|Your Website|Provided design|
|-|-|
|![Screenshot_20230524_055412](https://github.com/MohamedRoshdy2021/Conference-project/assets/25227022/b3d306fc-c65b-47d6-9311-0ec0ed8a29a4)|![Screenshot_20230524_055627](https://github.com/MohamedRoshdy2021/Conference-project/assets/25227022/c8a40685-dbdd-48f3-a82b-48aca8accf12)|